src/Arcus.Security.Core/Providers/MutatedSecretNameSecretProvider.cs

```diff
@@ -190,8 +215,14 @@ protected async Task<T> SafeguardMutateSecretAsync<T>(string secretName, Func<st
         /// <exception cref="ArgumentException">Thrown when the <paramref name="afterMutation"/> is <c>null</c>.</exception>
         protected T SafeguardMutateSecret<T>(string secretName, Func<string, T> afterMutation)
         {
-            Guard.NotNullOrWhitespace(secretName, nameof(secretName), "Requires a non-blank secret name when mutating secret names");
-            Guard.NotNull(afterMutation, nameof(afterMutation), "Requires a function to run after the secret name mutation");
+            if (string.IsNullOrWhiteSpace(secretName))
+            {
+                throw new ArgumentNullException(nameof(secretName), "Requires a non-blank secret name when mutating secret names");
+            }
+            if (afterMutation is null)
```

Comment:
```suggestion

            if (afterMutation is null)
```
Best to have a blank line between curly brackets, following the Microsoft code style.